src/components/specific/files/file-uploader/FileUploader.vue

```diff
@@ -88,13 +93,16 @@ export default {
     const fileUploads = ref([]);
 
     const uploadFiles = event => {
+      const targetFiles = Array.from(event.target.files);
+      if (targetFiles.length === 0) return;
+
       let files = null;
-      if (event.dataTransfer) {
+      if (targetFiles.dataTransfer) {
         // Files from drag & drop
-        files = Array.from(event.dataTransfer.files);
+        files = Array.from(targetFiles.dataTransfer.files);
```

Comment:
The drag & drop is broken here...
As Hugo pointed out, I suggest to remove the files length check above and just keep the `event` variable as before.
Let me know if I'm missing something.